client_handler.cpp

```diff
@@ -1785,6 +1785,10 @@ bool ClientHandler::OnSelectClientCertificate(
 			return false;
 		}
 
+#pragma warning(push, 0)

```

Comment:
```suggestion
#pragma warning(push)
```

It seems that push N may be redundant.

Reply:
I have fixed it.